src/MadLibEntry/MadLibEntry.js

```diff
@@ -21,15 +21,22 @@ const MadLibEntry = ({ madLib }) => {
       str = str.replaceAll(key, userInputs[key])
     })
     console.log(str)
-    setResult(str)
+    setResult({result: str, isFavorited: false})
```

Comment:
Is the `isFavorited` property something that will be used to conditionally-render something later on?

Reply:
Yes! Its used to conditionally render the style of the favorited button.